### ISSUE TITLE
Set filter property value as variable

### ISSUE THREAD

**Issue by erickskrauch** [WarmUpTill/SceneSwitcher#1232]
### Background

I need to set the filter's property from calculated variable.a 

### Solution

Add an option to set the value as variable.a 

### Alternatives

I see there is a "macros property" option, but I can't understand how to set it, so it's not quite a solution :(

**Comment by Destroy666x** [comment]
What exactly do you need to do? Describing the problem accurately would help. This is an example of an action that you need to add:
![image](https://github.com/user-attachments/assets/36622bd8-c18d-44c7-98c4-c40c6fe06882)

It's not too clear that you need to use the `${}` syntax, but all the text fields have a tooltip specifying it. I was thinking about adding some variable inserter related to these text fields a while ago, not too sure if that's reliably doable without too much clutter though. The best I could think of was adding an icon next to fields that brings a popup with variables, but then almost all fields support variables now, so the UI would get quite spammy. And anything else like right click, keyboard shortcut, etc. to bring the popup is as missable as the tooltip.

**Comment by erickskrauch** [comment]
Yeah, that's literally what I needed. Ever since I found that there are separate actions for dealing with variables, like setting a fixed value or appending to the end of a string, I've never paid attention to the tooltip that there is a `${}` syntax. This really simplifies all my scripts and allows me to use variables wherever I needed them.

I apologize for diverting you to this silly question due to my inattention and thank you so much for already providing everything in your plugin. It's awesome and has allowed me to take my broadcasts to the next level!
